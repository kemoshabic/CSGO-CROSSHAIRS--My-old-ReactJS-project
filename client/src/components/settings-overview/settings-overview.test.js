import React from "react";
import { shallow } from "enzyme";
import Settings from "./settings-overview.component";

describe("Settings component", () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      settings: {
        headset: "text",
      },
    };

    wrapper = shallow(<Settings {...mockProps} />);
  });

  it("should render Settings component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should return expect value in displayed text", () => {
    const text = wrapper.find(".overview__text").at(0).text();
    expect(text).toEqual("text");
  });
});
