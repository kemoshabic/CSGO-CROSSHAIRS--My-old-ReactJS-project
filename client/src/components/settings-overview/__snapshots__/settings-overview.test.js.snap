// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings component should render Settings component 1`] = `ShallowWrapper {}`;
